C-Plus-Plus/dp/Minimum_Sum_Partition.cpp

```diff
@@ -0,0 +1,75 @@
+// Minimum Sum Partition
+#include <iostream>
+#include <string>
+using namespace std;
+
+// Partition the Set into two subsets Set1, Set2 such that the
```

Comment:
@sharur7 Convert all single-line comments into one multiline comment. Something like this
```
/* 
...
*/
```

Reply:
Done @iamrajiv